### PR TITLE
Feature/signincomponent

### PR DESCRIPTION
**Pull request by mliu433**
#### Description
Add sign in components 
go to http://localhost:3000/auth to view the login page
#### Review Checklist
- [x] The requirements on the description have been met
- [ ] The relevant documentation has been updated:
    - [ ] code comments for classes / methods / complex statements
    - [ ] package readme
    - [ ] documentation for high level approaches or product info
- [x] Tests:
    - [ ] new tests written
    - [ ] existing affected tests updated
    - [ ] all green and passing
- [ ] Tested by reviewer

### CODE REVIEW

**Comment by Cvos0162** [comment]
image
Im getting this page with weird color theming.
wondering if its just me

File: app/src/components/LoginInput.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        password={password}
+        setPassword={setPassword}
+        placeholderTextForEmail="Enter Email"
+        placeholderTextForPassword="Enter Password"
+        loading={false} //TODO: backend loading request
+        submit={handleLogin}
+        submitText="Login"
+      />
+    </React.Fragment>
+  );
```
Comment (Cvos0162) [inline app/src/components/LoginInput.tsx:53]: Indentation here. gotta be aligned with return

File: app/src/pages/LoginPage.tsx [hunk 2]
```diff
@@ -26,16 +28,20 @@ const Page: React.FC = () => {
...
+    
+      <IonContent >
         <IonHeader collapse="condense">
           <IonToolbar>
             <IonTitle size="large">{name}</IonTitle>
           </IonToolbar>
         </IonHeader>
-        <ExploreContainer name={name} />
+        <div className="container">
+        <LoginInput />
```
Comment (Cvos0162) [inline app/src/pages/LoginPage.tsx:39]: Indentation

File: app/src/components/LoginInput.tsx (hunk 1 above)
Comment (mliu433) [reply app/src/components/LoginInput.tsx:53]: did you do npm install?

**Comment by mliu433** [comment]
> image
> Im getting this page with weird color theming.
> wondering if its just me

**Review by xoneon — APPROVED** [review]
Looks good

**Pull request merged**